framework_tvb/tvb/adapters/exporters/export_manager.py

```diff
@@ -130,9 +155,14 @@ def export_data(self, data, exporter_id, project):
         try:
             data_export_folder = self._build_data_export_folder(data)
 
-            # Here is the real export                    
-            self.logger.debug("Start export of data: %s" % data.type)
-            export_data = exporter.export(data, data_export_folder, project)
+            self.copy_dt_to_export_folder(data, data_export_folder)
+            if data_export_folder is not None and len(os.listdir(data_export_folder)) > 1:
```

Comment:
Instead of this check, wound't be better to have a new button in web GUI "Export with references"  to differentiate between the 2 scenarios ?
In case a user wants to export a TimeSeries and only use it in Matlab, for example, a simple export (old form) could work better